### ISSUE TITLE
Feature Discoverability

### ISSUE THREAD

**Issue by pfitzseb** [julia-vscode/julia-vscode#1243]
Juno has both it's own menu entry and puts a bunch of buttons into the global toolbar:
![image](https://user-images.githubusercontent.com/6735977/82998257-cba49e80-a007-11ea-92a8-4091d6c20b2b.png)
It also adds a few entries to the context menu (e.g. in the editor):
![image](https://user-images.githubusercontent.com/6735977/82998379-f2fb6b80-a007-11ea-9c1c-696e2885add2.png)

AFAICT, the first two options are basically impossible in VSCode, but we _can_ add context menu entries. Apart from that this issue isn't particularly actionable, but we should definitely think about how to improve feature discoverability.

**Comment by davidanthoff** [comment]
I guess another important part is docs... They are mostly non-existent. In theory they are hosted at https://github.com/julia-vscode/docs.

**Comment by pfitzseb** [comment]
Yep. I kinda like what we did with https://junolab.org/ -- ideally we could do something similar for VSCode and add the command names/keybindings as well.

**Comment by davidanthoff** [comment]
Yes, the Juno homepage is great!

I think I have some GSoD emails in my inbox, need to take a look, maybe that can help :)

**Comment by waldyrious** [comment]
> Juno has both it's own menu entry and puts a bunch of buttons into the global toolbar

I believe extensions can add items to the [Activity Sidebar](https://code.visualstudio.com/docs/getstarted/userinterface) — IIRC GitLens does that.
